wp-content/themes/launchframe/templates/home.twig

```diff
@@ -12,7 +12,7 @@
 	<section class="u-bg__color--gray-lighter">
 		<div class="container">
 			<div class="row">
-				<article>
+				<article class="col-sm">
```

Comment:
Is this an in-use selector? If it's a single column you can denote that with col